src/app/util/ember-compatibility-functions.cpp

```diff
@@ -84,5 +86,11 @@ void ResetEmberAfObjects()
 }
 
 } // namespace Compatibility
+
+CHIP_ERROR CheckIfClusterCommandExists(chip::ClusterId aClusterId, chip::CommandId aCommandId, chip::EndpointId aEndPointId)
```

Comment:
This method doesn't seem to do what it claims to do. Are you going to change it in the future? If so, please add some "TODO" comment.

Reply:
Added a TODO comment.
@mrjerryjohns I remembered that your design for Command dispatch / Encoding covered this part (Endpoint - Cluster - Command catalog)?